### PR TITLE
Add App Insights to Building Real-World Cloud Apps e-book #3873

### LINKED ISSUE #3873: Add App Insights to Building Real-World Cloud Apps e-book
The telemetry chapter of the Building Real-World Cloud Apps e-book was written before App Insights was released.

https://docs.microsoft.com/en-us/aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/monitoring-and-telemetry

### PR DESCRIPTION
**Pull request by isaac2004**
Fixes #3873

Added note regarding application insights to telemetry doc

### CODE REVIEW

**Comment by isaac2004** [comment]
@Rick-Anderson

File: aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/monitoring-and-telemetry.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ A lot of people rely on customers to let them know when their application is dow
 
 ## Buy or rent a telemetry solution
 
+> [!NOTE]
+> This article was written before [Application Insights](https://azure.microsoft.com/services/application-insights/) was released. Application Insights is the preferred approach for telemetry solutions on Azure. See [Set up Application Insights for your ASP.NET website](https://docs.microsoft.com/azure/application-insights/app-insights-asp-net) for more information
```
Comment (scottaddie) [inline aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/monitoring-and-telemetry.md]: Please add a period at the end of the last sentence.

**Comment by Rick-Anderson** [comment]
@isaac2004  Really great job on this. Much appreciated.

**Comment by scottaddie** [comment]
Nice work as always, @isaac2004!

**Pull request merged**